ci-operator/step-registry/upi/install/libvirt/upi-install-libvirt-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: upi-install-libvirt
+  steps:
+  #- ref: upi-install-rbac
```

Comment:
Nit: Should we commit commented-out steps?

Reply:
The intent was to leave a placeholder example for what the upi chain will look like when complete. Since the libvirt install chain is invalid as it exists, I don't think it matters either way. If it's more disruptive to your work with the domain creation and initial job bootstrapping, I don't mind keeping the chain barebones for now.

Reply:
I was only asking in case we want to remove a comment that isn't providing information.  No worries from me either way.